Clava-JS/src-api/clava/events/Events.ts

```diff
@@ -0,0 +1,28 @@
+import { Joinpoint } from "../../Joinpoints.js";
```

Comment:
This file can be moved to the lara-framework repo. It needs to have a generic parameter for the join point that extends LaraJoinPoint, then the weaver generator can specialize to the join point of the specific compiler it is generating code for.